charts/camunda-platform/test/integration/scenarios/lib/chart-upgrade-taskfile.yaml

```diff
@@ -8,7 +8,17 @@ vars:
 tasks:
   pre:
     cmds:
-    - echo "No pre task for this test."
+    # TODO: Remove it once 8.5 cycle is done.
```

Comment:
@jessesimpson36 The needed upgrade steps are mentioned here (will be added to the official docs when all refactors are done for the parent epic).